### PR TITLE
White labelling Base commit

### PR DESCRIPTION
**Pull request by ashwin1111**
## Summary by CodeRabbit

- **New Features**
  - Dynamic branding capabilities introduced, allowing customization of brand-specific elements throughout the application.
  - Theme customization support added with new color variables for consistent branding.

- **Enhancements**
  - Enhanced redirection links to be brand-specific, improving user experience for integrated services.
  - Email support functionality now retrieves brand-specific support email addresses.

- **Bug Fixes**
  - Fixed issues with hardcoded references to "Fyle," replacing them with dynamic brand names.
  - Corrected email links to use the appropriate brand support email.

- **Refactor**
  - Removed obsolete enums and hardcoded URLs, replacing them with dynamic brand configurations.
  - Streamlined the initialization of branding configurations during app startup.

- **Style**
  - Introduced a new "fyle" theme with custom CSS variables for UI elements.

- **Documentation**
  - Updated HTML templates to reflect dynamic branding changes in user-facing text.

- **Chores**
  - Updated `.gitignore` to include new branding configuration files.
  - Implemented scripts for setting up branding configurations based on environment variables.

- **Tests**
  - Added test suites for the new `BrandingService` to ensure proper initialization and functionality.

### CODE REVIEW

**Comment by ashwin1111** [comment]
@coderabbitai review

**Comment by ashwin1111** [comment]
@coderabbitai review

File: src/app/shared/components/si/configuration/configuration-export-settings/configuration-export-settings.component.ts [hunk 1]
```diff
@@ -4,8 +4,9 @@ import { AbstractControl, FormBuilder, FormGroup, ValidatorFn, Validators } from
...
+import { CCCExpenseState, ConfigurationCta, CorporateCreditCardExpensesObject, FyleField, ExpenseGroupedBy, ExpenseState, ExportDateType, IntacctReimbursableExpensesObject, ExpenseGroupingFieldOption, Page, ToastSeverity, IntacctOnboardingState, ProgressPhase, IntacctUpdateEvent, AppName, IntacctExportSettingDestinationOptionKey } from 'src/app/core/models/enum/enum.model';
 import { ExportSettingDestinationAttributeOption, IntacctDestinationAttribute, PaginatedintacctDestinationAttribute } from 'src/app/core/models/si/db/destination-attribute.model';
 import { ExportSettingFormOption, ExportSettingGet, ExportSettingModel, ExportSettingOptionSearch } from 'src/app/core/models/si/si-configuration/export-settings.model';
 import { IntegrationsToastService } from 'src/app/core/services/common/integrations-toast.service';
```
[the remark this answers is not in the document]
Comment (ashwin1111) [reply src/app/shared/components/si/configuration/configuration-export-settings/configuration-export-settings.component.ts:12]: is there any linting rule to fix this automatically?

File: src/app/shared/components/si/configuration/configuration-advanced-settings/configuration-advanced-settings.component.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { Component, OnInit, ViewChild } from '@angular/core';
 import { AbstractControl, FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { Router } from '@angular/router';
 import { forkJoin } from 'rxjs';
-import { AppName, ConfigurationCta, FyleField, IntacctLink, IntacctOnboardingState, IntacctReimbursableExpensesObject, IntacctUpdateEvent, Page, PaymentSyncDirection, ProgressPhase, ToastSeverity } from 'src/app/core/models/enum/enum.model';
+import { AppName, ConfigurationCta, FyleField, IntacctOnboardingState, IntacctReimbursableExpensesObject, IntacctUpdateEvent, Page, PaymentSyncDirection, ProgressPhase, ToastSeverity } from 'src/app/core/models/enum/enum.model';
 import { EmailOptions } from 'src/app/core/models/qbd/qbd-configuration/advanced-setting.model';
 import { AdvancedSetting, AdvancedSettingFormOption, AdvancedSettingsGet, AdvancedSettingsPost, HourOption } from 'src/app/core/models/si/si-configuration/advanced-settings.model';
 import { IntegrationsToastService } from 'src/app/core/services/common/integrations-toast.service';
```
[the remark this answers is not in the document]
Comment (ashwin1111) [reply src/app/shared/components/si/configuration/configuration-advanced-settings/configuration-advanced-settings.component.ts:8]: where is `ConfigurationCta` removed? I've removed only `IntacctLink` in the above code no?

**Review by ruuushhh — APPROVED** [review]

**Review by labhvam5 — APPROVED** [review]

**Pull request merged**